Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LightWeightExtensionSameplePage.xaml

```diff
@@ -0,0 +1,508 @@
+﻿<Page x:Class="Uno.Gallery.Views.SamplePages.LightWeightExtensionSameplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid>
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.MaterialTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<!-- Button -->
+						<smtx:XamlDisplay UniqueKey="LightweightExtensionSamplePage_Button_Material"
+										  smtx:XamlDisplayExtensions.Description="The examples contrast ordinary Buttons with others that apply an Extension to override some properties, as defined in the Material/Buttons.xaml ResourceDictionary."
+										  smtx:XamlDisplayExtensions.IgnorePath="XamlDisplay\StackPanel">
+							<StackPanel Spacing="40">
+
+								<Button Content="Default Button Style"
+										Style="{StaticResource MaterialFilledButtonStyle}" />
+
+								<Button Content="Overridden Button Style"
+										BorderThickness="2"
+										Style="{StaticResource OverridenMaterialButton}" />
+
+								<!-- OverridenMaterialButton defintion: -->
+								<!--<Style x:Key="OverridenMaterialButton"
+	 BasedOn="{StaticResource MaterialFilledButtonStyle}"
+	 TargetType="Button">
+<Setter Property="local:ControlExtensions.ResourcePath"
+		Value="ms-appx:///Views/ControlResourcesOverride/Material/Button.xaml" />
+</Style>-->
+
+								<Button Content="Default Button Style - Disabled"
+										IsEnabled="False"
+										Style="{StaticResource MaterialFilledButtonStyle}" />
+
+								<Button Content="Overridden Button Style - Disabled"
+										BorderThickness="2"
+										IsEnabled="False"
+										Style="{StaticResource MaterialFilledButtonStyle}"
+										local:ControlExtensions.ResourcePath="ms-appx:///Views/ControlResourcesOverride/Material/Button.xaml" />
+
+								<!-- Material/Button.xaml content: -->
+								<!--<SolidColorBrush x:Key="FilledButtonForeground"
```

Comment:
just use oneliner syntax for the resources
in the xamlstyler settings, add SolidColorBrush into "Newline exemption elements"